web/index.html

```diff
@@ -128,6 +128,12 @@
     return !isNaN(parseFloat(n)) && isFinite(n);
 }
 
+function selectInputText(input) {
+	setTimeout(function() {
```

Comment:
Have you tested if replacing setTimeout with requestAnimationFrame works as well? Would avoid that pesky hard coded value :)